learn-patcher.xyz/patches.md

```diff
@@ -5,32 +5,50 @@ coverY: 0
 
 # Patches
 
-![](<../.gitbook/assets/patcher\_promo\_280122 (1) (Small).png>)
```

Comment:
revert this line

Reply:
revert to original:
![](<../.gitbook/assets/patcher\_promo\_280122 (1) (Small).png>)